packages/sync/src/Functions.php

```diff
@@ -222,6 +290,11 @@ public static function home_url() {
 		return esc_url_raw( apply_filters( 'jetpack_sync_home_url', $url ) );
 	}
 
+	/**
+	 * Return the siteurl.
```

Comment:
Should we clarify that this site URL is in escaped form?

Reply:
ecbf5af85